examples/favorites_advanced/getx_favorites_advanced/lib/base/resources/color_styles.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:flutter/material.dart';
```

Comment:
@RomanovaPrime, can you please explain why we need custom colors on the project level?

Reply:
Because we are going to use them in different parts in our project. If we decide to change them, it will be easier to do it in only one place, like a folder 'resources'.

Reply:
Relocated in favorites_advanced_base